cockroachcloud/cluster-management.md

```diff
@@ -191,7 +191,7 @@ All databases are not backed up at the same time. Each database is backed up eve
 
 {{ site.data.products.dedicated }} advanced clusters have a **PCI ready** panel to monitor the status of security features required for [PCI readiness](pci-dss.html). Feature statuses will update from **INACTIVE** to **ACTIVE** once you configure them. Learn more about configuring these features:
 
-- [Audit logs](cloud-org-audit-logs.html)
+- [{{ site.data.products.db }} Organization Audit logs](cloud-org-audit-logs.html)
```

Comment:
These bullets should have a period per our style.

Reply:
I disagree in this case. There is not a single hard-and-fast rule about bullets and title vs sentence case; [it depends](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#lists). These are not sentences or sentence-like.

Reply:
My mistake, you're right. I totally forgot that our style book leaves this open.